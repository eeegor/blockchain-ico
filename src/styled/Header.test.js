import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Header } from '.';

describe('Header', () => {
	test('renders default header', () => {
		const tree = renderer.create(<Header>Content</Header>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
