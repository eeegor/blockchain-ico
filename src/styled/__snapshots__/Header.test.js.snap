// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders default header 1`] = `
.c0 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 12px;
  background: #fff;
  box-shadow: 0px 2px 3px 0px rgba(0,14,70,.1);
  z-index: 8000;
}

@media (min-width:767px) {
  .c0 {
    height: 84px;
  }

  .c0 .logo {
    width: 160px;
    height: 48px;
  }
}

<div
  className="c0"
>
  Content
</div>
`;
